### ISSUE TITLE
Missing translation strings in V8

### ISSUE THREAD

**Issue by l-delin** [PrestaShop/PrestaShop#30529]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

The strings for all sample files in the **Advanced Parameters > Import > Download sample csv files** menu are missing from the Crowdin catalog and cannot be translated. 
The catalog shows the strings in 1.6 up to 1.7.8, but the strings are missing in 8.0.0.

![image](https://user-images.githubusercontent.com/113511191/206407818-d74ef5cf-2995-4290-af1d-8ad71d76fe41.png)

Strings appear in 1.6 up to 1.7.8 but do not appear in 8.0.0. For example:
![image](https://user-images.githubusercontent.com/113511191/206409459-103431a5-d710-4581-80c3-489d02550ae2.png)
### Expected behavior

Strings should appear in the 8.0 Crowdin catalog for translation.

### Steps to reproduce

1. Use any localization pack on PS 8.0.0, 
2. Go to Advanced Parameters > Import > Download sample csv files
3. Check if the menu is translated.

### PrestaShop version(s) where the bug happened

8.0.0

**Comment by paulnoelcholot** [comment]
Hello,

I reproduce the issue with PrestaShop version 8.0.0, I'll add this to the backlog so it can be fixed.

Please be aware that some issues might take a very long time to be resolved. If this one is important to you and you cannot wait for it to be fixed on the project’s own time, we strongly suggest you consider [contacting a professional](https://www.prestashop-project.org/support/) to help you. 

If you fix the issue on your end, please [contribute it back to the project](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/). Remember that the more people contribute, the better PrestaShop becomes for everyone.

Tested en 8.0.0 ==> NOK
![image](https://user-images.githubusercontent.com/112863083/206419067-ae152962-c86a-4a2d-bd6e-c0639ccd0da7.png)


Tested en 1.7.8.7 ==> OK
![image](https://user-images.githubusercontent.com/112863083/206419151-ed28f805-0871-46f2-98ab-52f6d75787a2.png)


Thank you

**Comment by jolelievre** [comment]
Hi guys,

I checked three things:

1. https://github.com/PrestaShop/TranslationFiles/blob/master/8/fr-FR/AdminAdvparametersFeature.fr-FR.xlf#L1638 contains the wording and its translation
2. I also downloaded this file https://i18n.prestashop-project.org/translations/8/fr-FR/fr-FR.zip and I can find the appropriate wording in it.
3. I tested locally with french language and the label is translated (see screenshot)
![Capture d’écran 2023-03-20 à 18 25 51](https://user-images.githubusercontent.com/13801017/226419649-9af9739c-f845-446b-8a0c-f661b9440750.png)


Besides @paulnoelcholot even in your screenshot for 8.0 the label is translated, so it seems to me there's no bug related to this wording. @l-delin I suggest you reinstall your shop or update your language pack from the BO.
